docs/developing.rst

```diff
@@ -147,7 +147,7 @@ Rebuild static assets to make sure mockprock ui scripts are available. In devsta
 
    make dev.static.lms
 
-Then back in your host shell::
+Then back in your host shell, install and run the server. You may need to run this in a virtual environment.::
```

Comment:
I'd honestly go further here and say we highly recommend using a virtual environment